app/funkcje.js

```diff
@@ -0,0 +1,66 @@
+$(document).ready(function ()
+{
+    'use strict';
+    $(function ()
```

Comment:
$(function () is shorthand for $(document).ready(function () so you shouldn't use both